content/en/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information.md

```diff
@@ -210,6 +210,8 @@ variables and `downwardAPI` volumes:
   * A Container's CPU request
   * A Container's memory limit
   * A Container's memory request
+  * A Container's hugepages limit, available since v1.20 (if DownwardAPIHugePages feature gate enabled)
+  * A Container's hugepages request, available since v1.20 (if DownwardAPIHugePages feature gate enabled)
   * A Container's ephemeral-storage limit, available since v1.8.0-beta.0
   * A Container's ephemeral-storage request, available since v1.8.0-beta.0
```

Comment:
```suggestion
  * A Container's ephemeral-storage limit
  * A Container's ephemeral-storage request
```

Reply:
and I would also revise this to drop the other mentions of version support for long-graduated fields such as `metadata.uid`.